lte/gateway/python/integ_tests/common/subscriber_db_client.py

```diff
@@ -163,12 +222,35 @@ def list_subscriber_sids(self):
         sids = ['IMSI' + sid.id for sid in sids_pb]
         return sids
 
+    def add_apn_details(self, apn_data):
+        for apn in apn_data:
+            apn_config = self._get_apn_data(apn)
+            SubscriberDbGrpc._try_to_call(
+                lambda: self._subscriber_stub.AddApn(apn_config)
+            )
+
+    def list_apn_data(self):
```

Comment:
Does this retrieve only the apn names? Maybe then changing the name to list_apn_names would be better.